src/subnet_manager_driver.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
```

Comment:
I have read that it is preferred to include as many files as possible in the .cpp file to speed up compile times. Here is a short stack overflow discussion on the topic: http://stackoverflow.com/questions/3002110/include-in-h-or-c-cpp

It probably doesn't matter which way we do it, so long as we have a thought out reason and are consistent.